common/OpTestOpenBMC.py

```diff
@@ -515,11 +515,11 @@ def wait_for_runtime(self, timeout=10):
     '''
     Boot progress
     curl   -b cjar   -k  -H  'Content-Type: application/json'   -d '{"data": [ ] }'
-    -X GET https://bmc//org/openbmc/sensors/host/BootProgress
+    -X GET https://bmc//xyz/openbmc_project/state/host0/attr/BootProgress
     '''
     def old_wait_for_runtime(self, timeout=10):
         data = '\'{"data" : []}\''
-        obj = "/org/openbmc/sensors/host/BootProgress"
+        obj = "/xyz/openbmc_project/state/host0/attr/BootProgress"
```

Comment:
Add a new function `get_boot_progress` to actually get the current boot progress state and use that function in both `old_wait_for_runtime` and `old_wait_for_standby`. And also add that `get_boot_progress` API in the testcase.

Reply:
And can you have a look at the above comment.